src/components/pager/__internal__/pager-navigation.component.tsx

```diff
@@ -46,7 +46,7 @@ export interface PagerNavigationProps {
       | React.KeyboardEvent<HTMLButtonElement>
   ) => void;
   /** onPagination Callback triggered when a change is triggered */
-  onPagination: (pageSize: number, currentPage: number, origin: string) => void;
+  onPagination: (currentPage: number, pageSize: number, origin: string) => void;
```

Comment:
**issue**: I can understand that having these args ordered alphabetically is desirable, but unfortunately as this callback is already public facing by being documented and potentially already used in products this may cause some issues. If users have already passed args to the `onPagination` callback the number they have passed as the `pageSize` will now be the `currentPage` and vice versa - potentially causing unintentional behaviour.

Reply:
This applies to the suggested changes in the `Pager` component too

Reply:
Is there any particular reason why this change is needed? We'd be happy to discuss alternative solutions or consider it, if you feel its needed

Reply:
the reason for that change, if I'm not mistaken, is because the callback is called with the wrong arguments order. In the documentation (storybook, Pager interface, PagerNavigation interface) is:  `pageSize: number, currentPage: number, origin: string`
when in the code the `onPagination` callback is called with `(currentPage, pageSize, origin) => {}`, for instance [here](https://github.com/Sage/carbon/blob/master/src/components/pager/__internal__/pager-navigation.component.tsx#L101). Instead of changing the order of the arguments in the callback I thought changing the documentation will be less painful. Please let me know if i missed anything or anything is incorrect.

Reply:
@MarcServat thanks for pointing this out, I can see internally within `handlePageInputChange`  we are calling `onPagination` with the args in the manner you've described. I'll wait for a second reviewer to get their opinion on this, as I agree it would be good to document the args as they're actually used.

Reply:
I agree with the change @MarcServat has made here. This is an `internal` component too so we can make changes without it being classed as officially breaking. [RFC for internal stuff is here if you want to have a read on it](https://github.com/Sage/carbon/blob/master/rfcs/text/internal-interfaces.md)